OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -59,10 +59,9 @@ public abstract class SupportPowerInfo : ConditionalTraitInfo
 		public readonly string BeaconImage = "beacon";
 		[SequenceReference("BeaconImage")] public readonly string BeaconPoster = null;
 		[PaletteReference] public readonly string BeaconPosterPalette = "chrome";
-		[SequenceReference("BeaconImage")] public readonly string ClockSequence = "clock";
-
-		[SequenceReference("BeaconImage")] public readonly string ArrowSequence = "arrow";
-		[SequenceReference("BeaconImage")] public readonly string CircleSequence = "circles";
+		[SequenceReference("BeaconImage")] public readonly string ClockSequence = null;
+		[SequenceReference("BeaconImage")] public readonly string ArrowSequence = null;
+		[SequenceReference("BeaconImage")] public readonly string CircleSequence = null;
```

Comment:
This is an easy case to write a rule rule for (and there are examples you can copy).

pseudocode:
````
var node = FirstOrDefault(key == "ArrowSequence")
if (node == null)
     AddNode(new MiniYaml("ArrowSequence", "arrow"))
/* else do nothing, because something is already specified */
```